tests/api/test_landing_page.py

```diff
@@ -71,6 +71,7 @@ def test_schema(example_url, schema_url):
 
 def test_api_landing_page():
     LandingPage(
+        type="Catalog",
```

Comment:
This feels weird to me but stac-pydantic says the landing page model is a Catalog, while I'm not sure to read this in the documentation